compiler/src/dotty/tools/dotc/parsing/Parsers.scala

```diff
@@ -504,7 +504,7 @@ object Parsers {
     def selector(t: Tree): Tree =
       atPos(startOffset(t), in.offset) { Select(t, ident()) }
 
-    /** Selectors ::= ident { `.' ident()
+    /** Selectors ::= id { `.' id }
```

Comment:
Good commit. Possible further idea: mention `id` in the comment for `isIdent`, also given they're not consistent? So authors of other comments can also find the right terminal name to use in comments?
(Well, being consistent might be even better but I'm not sure what to pick).
But that shouldn't hold this PR.